code/AssetLib/3DS/3DSLoader.h

```diff
@@ -208,6 +208,15 @@ class Discreet3DSImporter : public BaseImporter {
     */
     void ReplaceDefaultMaterial();
 
+    bool ContainsTextures(unsigned int i) const {
+        return mScene->mMaterials[i].sTexDiffuse.mMapName.length() != 0 ||
+               mScene->mMaterials[i].sTexBump.mMapName.length() != 0 ||
+               mScene->mMaterials[i].sTexOpacity.mMapName.length() != 0 ||
+               mScene->mMaterials[i].sTexEmissive.mMapName.length() != 0 ||
+               mScene->mMaterials[i].sTexSpecular.mMapName.length() != 0 ||
+               mScene->mMaterials[i].sTexShininess.mMapName.length() != 0;
```

Comment:
```suggestion
        return mScene->mMaterials[i].sTexDiffuse.mMapName.empty() ||
               mScene->mMaterials[i].sTexBump.mMapName.empty() ||
               mScene->mMaterials[i].sTexOpacity.mMapName.empty() ||
               mScene->mMaterials[i].sTexEmissive.mMapName.empty() ||
               mScene->mMaterials[i].sTexSpecular.mMapName.empty() ||
               mScene->mMaterials[i].sTexShininess.mMapName.empty();
```
This is a bit more succinct and cleaner to read.

Reply:
Done!